package.json

```diff
@@ -18,7 +18,8 @@
   "dependencies": {
     "fs-plus": "^3.0.0",
     "grim": "^2.0.1",
```

Comment:
@jasonrudolph : I'm assuming that I don't need to increment the version of `metrics-reporter` in `providedServices` here since I'm just adding new methods and the old ones are still supported.  Is that the case?

Reply:
Since this is an additive change, I think it makes sense to increment the version from `1.0.0` to `1.1.0`.

Since the new methods didn't exist in version `1.0.0` of the service, and since you want a way for packages to consume the new methods, those packages need to be able to declare a dependency on a version of the service that includes the new methods. By incrementing the version to `1.1.0`, existing consumers (e.g., the [`welcome` package](https://github.com/atom/welcome/blob/7e504a6895f6123942ed6dbb219dd564604e5285/package.json#L19)) should continue to work, and packages that want to use the new methods can declare a minimum dependency of `1.1.0`.

Does that sound reasonable?

Reply:
got it, thanks!  Have incremented the `provideReporter` version here.